packages/cli/src/evaluation.ee/test-runner/test-runner.service.ee.ts

```diff
@@ -313,6 +301,34 @@ export class TestRunnerService {
 			},
 		};
 
+		if (
+			!(
+				config.get('executions.mode') === 'queue' &&
+				process.env.OFFLOAD_MANUAL_EXECUTIONS_TO_WORKERS === 'true'
+			) &&
+			data.executionData
+		) {
+			const nodeExecutionStack: IExecuteData[] = [];
+			nodeExecutionStack.push({
+				node: triggerNode,
+				data: {
+					main: [[{ json: {} }]],
+				},
+				source: null,
+			});
```

Comment:
Do we still need it? It was used only to provide the requestDataset flag to the trigger node.

Reply:
Yes, this is expected in regular execution mode, otherwise fails for different errors.